### PR TITLE
RESTWS-821: Add docs for AdministrationLinksResource2_0

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Related PR: [https://github.com/openmrs/openmrs-module-webservices.rest/pull/484](https://github.com/openmrs/openmrs-module-webservices.rest/pull/484)
Related issue: [https://issues.openmrs.org/browse/RESTWS-821](https://issues.openmrs.org/browse/RESTWS-821)

### CODE REVIEW

File: source/includes/AdministrationLinks/administration-links.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+## Administration Links Overview
+
+Every installed module can register its administration pages. They are accessible through Legacy UI at [openmrs/admin/index.htm](openmrs/admin/index.htm)
+
+The following endpoints allow users to return list of installed modules with their administration links.
+
+## Available Operations for Administration Links type
+
+1. [List Administration Links](#list-administration-links)
```
Comment (dkayiwa) [inline source/includes/AdministrationLinks/administration-links.md]: Fix the links here too.

Comment (BartlomiejRasztabiga) [reply source/includes/AdministrationLinks/administration-links.md]: @dkayiwa Done :)

File: source/includes/AdministrationLinks/administration_links.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+```response
+{
+    "results": [
+        {
+            "uuid": "webservices.rest",
+            "display": "webservices.rest.title",
+            "title": "webservices.rest.title",
+            "administrationLinks": {
+                "module/webservices/rest/settings.form": "webservices.rest.manage.settings",
```
Comment (dkayiwa) [inline source/includes/AdministrationLinks/administration_links.md]: I guess we are gonna change these to reflect the actual values pointed to by the properties.

Comment (BartlomiejRasztabiga) [reply source/includes/AdministrationLinks/administration_links.md]: Sure, I've just updated the PR. I will be able to revert that change if we decide to go with message keys instead.

**Pull request merged**